nautobot/virtualization/filters.py

```diff
@@ -29,12 +32,28 @@
 
 
 class ClusterTypeFilterSet(NautobotFilterSet, NameSlugSearchFilterSet):
+    clusters = django_filters.ModelMultipleChoiceFilter(
```

Comment:
Should all of these added reverse FK filters be `NaturalKeyOrPKMultipleChoiceFilter` instances instead?

Reply:
Yes, this should be `NaturalKeyOrPKMultipleChoiceFilter(natural_key="name")` but will need to be changed to `NaturalKeyOrPKMultipleChoiceFilter(to_field_name="name")` after #1981 gets merged